coops-ui/src/views/AnnualReport.vue

```diff
@@ -234,7 +250,10 @@ export default {
       filingPaying: false,
       haveChanges: false,
       saveErrors: [],
-      saveWarnings: []
+      saveWarnings: [],
+
+      // Constant properties
+      EntityTypes
```

Comment:
Severin: Is this a constant, a property, a type or an enum? This has been called all of these :) May I suggest a consistent label/comment for this?

Cameron: This could definitely be more clear, good catch. Will update this!

Reply:
Fixed, thanks.